### DISCUSSION TITLE
大大加油

### DISCUSSION THREAD

**Discussion by se7enmuting** [husandro/WoWTools#3] [General]
大大加油，一直用你的 https://wago.io/he271Sq-s 这个WA，最近发现会失效，期待你的插件！！！

**Comment by husandro** [comment]
> 大大加油，一直用你的 https://wago.io/he271Sq-s 这个WA，最近发现会失效，期待你的插件！！！

如果, 按钮无效可以试试
使用了命令 /run SetCVar("ActionButtonUseKeyDown", 1)
